R/options_plumber.R

```diff
@@ -93,22 +93,21 @@ options_plumber <- function(
 }
 
 #' Get an option value, alternatively look in environment for value
-#' @rdname options_plumber
+#' @describeIn options_plumber DOCS
```

Comment:
Going to revert this in favor of future PR using `{options}`.

```suggestion
#' @rdname options_plumber
```